reproducibility_project/spe_subproject/summarize.py

```diff
@@ -0,0 +1,162 @@
+"""Summarize spe data and save as csv files."""
+import os
+
+import numpy as np
+import pandas as pd
+import signac
+import unyt as u
+
+
+def K_to_kj_per_mol_conversion(value):
+    """Convert energy unit from K to kj/mol."""
+    conversion_factor = (1 * u.Kelvin).to_value(
+        "kJ/mol", equivalence="thermal"
+    )  # conversion_K_to_kj_per_mol
+    converted = value * conversion_factor
+    return converted
+
+
+if __name__ == "__main__":
+    project = signac.get_project()
+
+    engines = ["lammps-VU", "hoomd", "gromacs", "mcccs", "gomc", "cassandra"]
+
+    molecules = [
+        "methaneUA",
+        "pentaneUA",
+        "benzeneUA",
+        "waterSPCE",
+        "ethanolAA",
+    ]
+    properties = [
+        "potential_energy",
+        "tot_vdw_energy",
+        "tail_energy",
+        "tot_electrostatics",
+        "short_range_electrostatics",
+        "long_range_electrostatics",
+        "tot_pair_energy",
+        "bonds_energy",
+        "angles_energy",
+        "dihedrals_energy",
+        "tot_bonded_energy",
+        "intramolecular_energy",
+        "intermolecular_energy",
+    ]
+
+    spe_data = dict()
+    for molecule in molecules:
+        spe_data[molecule] = dict()
+        for engine in engines:
+            spe_data[molecule][engine] = dict()
+            for property in properties:
+                spe_data[molecule][engine][property] = 0
+
+    # Parse LAMMPS SPE data
+    lmp_map = {
```

Comment:
I updated my lammps log-spe.txt such that we have 1:1 mapping. i.e.
"tot_pair_energy": "tot_pair_energy",
"tot_vdw_energy":"tot_vdw_energy",
"long_range_electrostatics": "long_range_electrostatics"